crystal/grip/shard.yml

```diff
@@ -11,6 +11,6 @@ targets:
 dependencies:
   grip:
     github: grkek/grip
```

Comment:
Could you please keep the `~>` ?

Reply:
With the `~>` other PR's will break as well because the new version is way different than the old Grip.

Reply:
ah, you mean that **breaking changes** will occurs on `0.26.6`
understood

but if I act like that, please make update on each `grip` release